File: baekjoon_OnlineJudge/BasicPractice/2609.py
import sys

input = lambda : sys.stdin.readline().rstrip()

def gcd(a, b):
    return gcd(b, a % b) if b else a
def lcm(a, b):
    return a * b // gcd(a, b)

A, B = map(int, input().split())
print(gcd(A,B))
print(lcm(A,B))

File: baekjoon_OnlineJudge/LearnCpp/2609.cpp
#include <iostream>

using namespace std;

int gcd(int a, int b) {
	int c;
	while (b != 0) {
		c = a % b;
		a = b;
		b = c;
	}
	return a;
}

int lcm(int a, int b) {
	return a * b / gcd(a, b);
}

int main() {
	int A, B;
	cin >> A >> B;
	cout << gcd(A, B) << '\n';
	cout << lcm(A, B) << '\n';
}